src/dodal/plans/wrapped.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, List, Mapping, Optional, Union
+
+import bluesky.plans as bp
+from bluesky.protocols import Readable
+
+from dodal.common import MsgGenerator
+
+"""
+Wrappers for Bluesky built-in plans with type hinting and renamed metadata
+"""
+
+
+def count(
+    detectors: List[Readable],
+    num: int = 1,
+    delay: Optional[Union[float, List[float]]] = None,
+    metadata: Optional[Mapping[str, Any]] = None,
+) -> MsgGenerator:
+    """
+    Take `n` readings from a device
+
+    Args:
+        detectors (List[Readable]): Readable devices to read
+        num (int, optional): Number of readings to take. Defaults to 1.
+        delay (Optional[Union[float, List[float]]], optional): Delay between readings.
+                                                               Defaults to None.
+        metadata (Optional[Mapping[str, Any]], optional): Key-value metadata to include
+                                                          in exported data.
+                                                          Defaults to None.
+
+    Returns:
+        MsgGenerator: _description_
+
+    Yields:
+        Iterator[MsgGenerator]: _description_
+    """
+    plan_args = (
+        {  # If bp.count added delay to plan_args, we could remove all md handling
```

Comment:
Should: Personally, I don't like comments like this. What information is it actually giving a future developer? Is this body of work going to be done? Should it be done? If so make an issue for it and link the issue here saying we can remove this code when the issue is complete, at least then the developer has an action they are able to do.

Reply:
Adjusted to reference a PR I've opened on Bluesky with the required change.